must-gather/collection-scripts/gather

```diff
@@ -1,12 +1,27 @@
 #!/bin/bash
 
+
+function is_hypershift() {
+  control_plane_topology=$(/usr/bin/oc get infrastructure cluster -ojson | jq .status.controlPlaneTopology)
+  if [[ "${control_plane_topology}" == "External" ]]; then
+    return 0
+  fi
+  return 1
+}
+
 # generate /must-gather/version file
 . version
 echo "numaresources-operator/must-gather" > /must-gather/version
 version >> /must-gather/version
 
 # machine/node resources
-resources+=(nodes machineconfigs machineconfigpools featuregates kubeletconfigs)
+resources+=(nodes featuregates)
+
+if is_hypershift; then
+  resources+=(configmaps)
```

Comment:
all of them?

Reply:
I mean, on which namespace?

Reply:
fixed that. configmap will be collected only from `openshift-config-managed` namespace which suppose to contain the kubeletconfig configmap

Reply:
yes. should be here:https://github.com/openshift-kni/numaresources-operator/pull/1234/files#diff-2257fba988165a6d7631558cbd11f1c7aafb71d06f02e10d2fd4e210519f7c4eR21